data/ecosystems/d/departurelabsic.toml

```diff
@@ -0,0 +1,51 @@
+title = "Departure Labs"
```

Comment:
duplicate of https://github.com/electric-capital/crypto-ecosystems/blob/master/data/ecosystems/d/departure-labs.toml
can remove this file